_includes/pbc/all/install-features/202304.0/install-the-gift-cards-feature.md

```diff
@@ -0,0 +1,1338 @@
+
+
+This document describes how to integrate the [Gift Cards feature](/docs/pbc/all/gift-cards/{{site.version}}/gift-cards.html) into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Gift Cards feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME   | VERSION | INTEGRATE GUIDE |
+| --- | --- | --- | 
+| Spryker Core | {{site.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html) | 
+| Cart | {{site.version}} |[Install the Cart feature](/docs/pbc/all/cart-and-checkout/{{site.version}}/install-and-upgrade/install-features/install-the-cart-feature.html)|
+|Product  | {{site.version}} |[Product feature integration](/docs/pbc/all/product-information-management/{{site.version}}/install-and-upgrade/install-features/install-the-product-feature.html)|
+|Payments  | {{site.version}} |[Payments feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/payments-feature-integration.html)|
+| Shipment | {{site.version}} |[Integrate the Shipment feature](/docs/pbc/all/carrier-management/{{site.version}}/install-and-upgrade/integrate-the-shipment-feature.html)|
+| Order Management | {{site.version}} |[Order Management feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/order-management-feature-integration.html)|
+| Mailing &amp; Notifications | {{site.version}} |[Mailing &amp; Notifications feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/mailing-and-notifications-feature-integration.html)|
+| Promotions &amp; Discounts | {{site.version}} |[Integrate the Promotions &amp; Discounts feature](/docs/pbc/all/discount-management/{{site.version}}/install-and-upgrade/integrate-the-promotions-and-discounts-feature.html)|
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/gift-cards:"{{site.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+| --- | --- |
+| CartCode | vendor/spryker/cart-code |
+| CartCodeExtension | vendor/spryker/cart-code-extension |
+| GiftCard | vendor/spryker/gift-card |
+| GiftCardBalance | vendor/spryker/gift-card-balance |
+| Nopayment | vendor/spryker/nopayment |
+
+{% endinfo_block %}
+
+### 2) Set up configuration
```

Comment:
As this section is blank and does not have any steps, just sub-steps, we can get rid of it
```suggestion
### 2) Set up the Gift Card purchase process
```